api_volontaria/apps/user/tests/tests_view_users.py

```diff
@@ -28,11 +22,9 @@ class UsersTests(CustomAPITestCase):
         'url',
         'id',
         'is_superuser',
-        'last_name',
         'is_active',
-        'first_name',
```

Comment:
Une raison specifique pour enlever `last_name` et `first_name` ?

Reply:
Oups, it's was missing in my serializers...